Topics/Software_Engineering/Software_Licenses.md

```diff
@@ -0,0 +1,134 @@
+# Software Licenses
+
+## Table of Contents
+1. [Introduction](#introduction)
+2. [Why are software licenses important?](#importance)
+3. [Types of Software Licenses](#license-types)
+    - [Public domain](#public-domain)
+    - [Permissive Licenses](#permissive-licenses)
+    - [Copyleft Licenses](#copyleft-licenses)
+        - [GNU Lesser General Public Licenses](#gnu-lesser-general-public-licenses)
+    - [Proprietary licenses](#proprietary-license)
+4. [How to choose the right license?](#how-to-choose-the-right-license)
+5. [Breach of Licenses](#breach-of-licence)
+6. [Conclusion](#conclusion)
+7. [Sources and References](#sources-and-references)
```

Comment:
Try to be consistent with your capitalization; some of your subsection titles follow traditional title capitalization rules, such as 'Types of Software Licenses', whereas others follow more phrasal conventions, such as 'How to choose the right license'. Also make sure that this matches the actual subheading in the doc!